test/Riok.Mapperly.IntegrationTests/Mapper/TestMapper.cs

```diff
@@ -62,6 +62,10 @@ public TestObjectDto MapToDto(TestObject src)
         [MapEnum(EnumMappingStrategy.ByName)]
         public partial TestEnumDtoByName MapToEnumDtoByName(TestEnum v);
 
+        [MapEnum(EnumMappingStrategy.ByName)]
+        [MapEnumValue(TestEnum.Value10, TestEnumDtoByName.Value10)]
```

Comment:
As the name matches, this does already work with the `ByName` strategy, doesn't it? Therefore it doesn't really test the `MapEnumValue` configuration. Adjust the test case by using different names or an enum with the same names, but different values and the `ByValue` strategy.

You may also need to update the integration test snapshots (see [integration tests](https://mapperly.riok.app/docs/contributing/tests#integration-tests) and [verify tests](https://mapperly.riok.app/docs/contributing/tests#verifytests) contribution documentation).